### DISCUSSION TITLE
Remove Transparency

### DISCUSSION THREAD

**Discussion by touhidulshawan** [isouravgope/Gruvbox-Kvantum#2] [General]
How can I remove transparency from the menubar. I tried but can not figure out yet.

![2022-09-18-024430_1366x768_scrot](https://user-images.githubusercontent.com/37136490/190875825-f90ba5da-33d1-4853-a08e-2a88c65ec565.png)

By the way, loved your work. Thanks for creating this work. ❤

**Comment by touhidulshawan** [comment]
@sourav2k

**Reply by TheSerphh** [reply]
> @sourav2k

@touhidulshawan

**Comment by TheSerphh** [comment]
![Screenshot from 2022-09-18 02-24-51](https://user-images.githubusercontent.com/103628008/190876113-55e1a933-5282-4458-873a-7b8d688e3ca6.png)


for me i use dolphin and i don't think the opacity is related to the kvantum theme , can you please apply a different theme and send a screenshot whether the opacity is still there or not, if it does i will check the svg for opacity and release a new tag

**Comment by touhidulshawan** [comment]
@sourav2k My bad. I have an option for menu in picom that triggered this issue. I fixed that now it looks fine

**Reply by TheSerphh** [reply]
> @sourav2k My bad. I have an option for menu in picom that triggered this issue. I fixed that now it looks fine

alright mate
